### PR TITLE
Fead: Login

### PR DESCRIPTION
**Pull request by Avikele**
Please review

### CODE REVIEW

File: src/Login.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+
+public class Login {
+    private static String password = "Admin";
+    private static String username = "admin";
+
+    static public void login() {
+        Scanner myObj = new Scanner(System.in);
+        System.out.println("Podaj username: ");
```
Comment (luszczyk597) [inline src/Login.java:10]: ponglisz xd

File: src/Login.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        String userName = myObj.nextLine();
+        System.out.println("Podaj password: ");
+        String password = myObj.nextLine();
+
+        if(userName == username && password == password) {
+            System.out.println("Zalogowano");
```
Comment (luszczyk597) [inline src/Login.java:16]: xd

**Comment by Avikele** [comment]
Repaired check it out

**Review by luszczyk597 — APPROVED** [review]
lgtm

**Pull request merged**